src/pages/_app.tsx

```diff
@@ -23,7 +23,7 @@ import {ReactQueryDevtools} from '@tanstack/react-query-devtools'
 import TrpcProvider from 'app/_trpc/Provider'
 
 import {PostHogProvider} from 'posthog-js/react'
-import PosthogClient from 'lib/posthog-client'
+import PostHogClient from 'lib/posthog-client'
```

Comment:
this could probably be `import posthog from 'lib/posthog-client'` since the file is exporting default variable